chat_server/README.md

```diff
@@ -1,27 +1,44 @@
 # Chat server
 
-This project was built arround two containers (yet only used for testing) :
-* A "web-container" with node.js and typescript
-* A "database-container" for postgress. It is used to store the table that imitates the one used in deployement.
+This is the chat service of the "portail". Here are some important points about it :
+1. It runs on the main database as the backend
+2. The authentification is done using jwt. The public key will be shared by both the backend and the chat server, but the only way to get it will be through an API request to the backend.
+
+## Setting up the project
 
-## Openning the project
+### Building the project locally
 
-You can run the project in two ways :
-1. Using the tools provided by VsCode (Remote containers)
-2. Using Linux and docker
+There are two steps you will need to do :
+1. Copy the environnement variables ;
+2. Create the database (on the same postgres instance as the backend).
+
+```bash
+sudo -i -u postgres
+psql
+CREATE DATABASE portail_message OWNER root
+```
 
 ### Building the container 
 
+Another way to use the project is with containers. If you are using vscode, I would recommand checking the "Remote container" extension, but you can still run :
```

Comment:
```suggestion
Another way to use the project is with containers. If you are using vscode, you should check the "Remote container" extension, but you can still run :
```

Prefer "you" to "I" when writing docs

Reply:
And even `You can also use containers. With Visual Studio Code, you may use the "Remote container" extension; otherwise, you can still run:"